tests/Foundation/FoundationCrawlerTraitJsonTest.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+use Illuminate\Http\Response;
```

Comment:
no need to import this, just write it in full like the rest of the file, no leading slash needed though